kafka_enforcer_common/src/main/java/com/tesla/data/enforcer/EnforceCommand.java

```diff
@@ -53,6 +57,8 @@
       arity = 1)
   protected int interval = 600;
 
+  private static final int TIMEOUT_MS = 30000;
```

Comment:
looks like this is only used for ZK, a more specific name and more information on what is the timeout for would be helpful

Reply:
removed ZK bits from this PR as not relevant to current changes, will keep this in mind